overlord/devicestate/devicestate_install_mode_test.go

```diff
@@ -85,6 +85,10 @@ func (s *deviceMgrInstallModeSuite) SetUpTest(c *C) {
 	classic := false
 	s.deviceMgrBaseSuite.setupBaseTest(c, classic)
 
+	s.AddCleanup(release.MockReleaseInfo(&release.OS{ID: "ubuntu"}))
+	// reload directory paths to match our mocked os-release
+	dirs.SetRootDir(dirs.GlobalRootDir)
```

Comment:
Should the cleanup set the SnapMountDir again so that it matches the OS? Although suites usually do it as part of their setup so it's probably fine not to

Reply:
no, I think it's a fair observation, even if it makes the code a little awkward